STYLE_GUIDE.rst

```diff
@@ -18,10 +22,7 @@ All files should start with the following::
 All imports should be absolute.
 
 
-All string formatting sound be done in the following style::
+All string formatting should be done in the following style::
```

Comment:
I actually wasn't sure if we should insist on `.format` or if we should support f-strings

Reply:
[pyupgrade](https://github.com/asottile/pyupgrade) will autoconvert to f-strings which a faster and easier to read.
https://www.scivision.dev/python-f-string-speed/